### ISSUE TITLE
Make some brading guidelines

### ISSUE THREAD

**Issue by GarthDB** [jadejs/jade#1926]
Put together a page on the website with branding guidelines and include logo downloads to go with the new logo (https://github.com/jadejs/jade/issues/1882).

**Comment by ForbesLindesay** [comment]
:+1:  I was thinking an "about" page, which could include a listing to give credit to some of the most prolific contributors/core maintainers, links to the gitter channel, twitter account, branding guidelines etc.

I'd also like to add react-jade and then-jade to the jade website, in their own sections with interactive documentation.  react-jade docs should be very similar in structure to the home page of http://facebook.github.io/react/ with interactive demos that show the react-jade code, the JavaScript code that generates, the HTML that is rendered, and finally the fully working application in a little box.

**Comment by GarthDB** [comment]
Sounds good.

**Comment by AhoyLemon** [comment]
I started a fork to try my hand at this. Gonna use the [GitHub Brand Assets](https://github.com/logos) as the general direction, but including a jade mixin so you can easily use the logos as inline SVG.

At the moment I'm using the SVGs in [this issue thread](https://github.com/jadejs/jade/issues/1882) as my resources. @GarthDB, if I could get the AI files from you, that would be helpful.

**Comment by GarthDB** [comment]
@AhoyLemon awesome! All the files are on [Pixelapse](https://www.pixelapse.com/GarthDB/projects/Jade/files).

**Comment by GarthDB** [comment]
Thinking through some website design ideas:

![screen shot 2015-06-26 at 11 04 53 am](https://cloud.githubusercontent.com/assets/125516/8382879/44822bc2-1bf3-11e5-81dd-b77091d075d0.png)

**Comment by GarthDB** [comment]
Someone recommended the http://www.firewatchgame.com/ site - could be interesting.

**Comment by AhoyLemon** [comment]
That is super duper pretty. Do you think that kind of approach makes sense for what is primarily a documentation website though.

From my own efforts, I've forked the branch, but I cannot make ~/docs/views build locally for me, so I've been stalled out a bit. Predominantly, my issue has been getting `unknown filter` errors on build.

**Comment by GarthDB** [comment]
@AhoyLemon take a look at https://github.com/jadejs/jade/issues/1927

**Comment by ForbesLindesay** [comment]
@GarthDB Looks totally awesome.  I'm hoping we can release this as a brand new site along with 2.0.0?

**Comment by GarthDB** [comment]
Sounds good. What's the timeline?



—

**Comment by TimothyGu** [comment]
@GarthDB there isn't a very strict timeline: 2.0.0 will *probably* be released when https://github.com/jadejs/jade/milestones/2.0.0 goes to 100% closed. Feel free to start working on the website right now, but don't feel obligated to do that.

**Comment by ForbesLindesay** [comment]
I've had a bit of a re-think of our timeline.  I've decided to defer re-working blocks and mixins to 3.0.0.  Based on this, 2.0.0 is close enough to ready to set a deadline.  On 1st November 2015 it will be released, unless there are completely show-stopping bugs.

**Comment by Berkmann18** [comment]
Is there any black and white or monochrome version of the logo?

**Comment by ForbesLindesay** [comment]
@Berkmann18 yes, check https://github.com/pugjs/pug-logo for all the variants of our logo
